deploy/gcp/main.tf

```diff
@@ -307,21 +327,28 @@ done
 helm install --namespace tidb-admin --name tidb-operator ${path.module}/charts/tidb-operator
 EOS
 
-    environment {
-      KUBECONFIG = "${local.kubeconfig}"
-    }
-  }
+
+environment = {
+KUBECONFIG = local.kubeconfig
+}
+}
 }
 
 resource "null_resource" "deploy-tidb-cluster" {
-  depends_on = ["null_resource.setup-env", "local_file.tidb-cluster-values", "google_container_node_pool.pd_pool", "google_container_node_pool.tikv_pool", "google_container_node_pool.tidb_pool"]
-
-  triggers {
-    values = "${data.template_file.tidb_cluster_values.rendered}"
-  }
+depends_on = [
+null_resource.setup-env,
+local_file.tidb-cluster-values,
+google_container_node_pool.pd_pool,
+google_container_node_pool.tikv_pool,
+google_container_node_pool.tidb_pool,
+]
+
+triggers = {
+values = data.template_file.tidb_cluster_values.rendered
+}
```

Comment:
These blocks dose not seem to be formatted properly

Reply:
I thought so too, but that's the output of `terraform fmt`

```bash
$ tail main.tf
done
EOS


environment = {
KUBECONFIG = local.kubeconfig
}
}
}

$ terraform fmt
$ tail main.tf
done
EOS


environment = {
KUBECONFIG = local.kubeconfig
}
}
}

```

:woman_shrugging:

Reply:
🤣